libs/contrib/Data/Elem.idr

```diff
@@ -0,0 +1,49 @@
+module Data.Elem
```

Comment:
I think this should be in `Data.List.Elem`, because similar structures make sense for other datastructures as well.